Gems/AWSGameLift/gem.json

```diff
@@ -11,10 +11,11 @@
     "user_tags": [
         "AWS",
         "Framework",
-        "Network"
+        "Network",
+        "SDK"
     ],
     "icon_path": "preview.png",
-    "requirements": "",
+    "requirements": "Users will need to enable Multiplayer Gem together to utilize AWSGameLift feature.",

```

Comment:
nit: "Users will need to enable the Multiplayer gem to support the AWSGameLift feature"

However, I don't see any other gem with the requirements filed set (https://github.com/o3de/o3de/search?l=JSON&p=11&q=requirements), is this expected to be a descriptive string?